robottelo/rhsso_utils.py

```diff
@@ -0,0 +1,119 @@
+"""Utility module to handle the rhsso-satellite configure UI/CLI/API testing"""
+import json
+import random
+
+from robottelo import ssh
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.config import settings
+from robottelo.constants import KEY_CLOAK_CLI
+from robottelo.constants import RHSSO_NEW_USER
+from robottelo.constants import RHSSO_RESET_PASSWORD
+from robottelo.datafactory import gen_string
```

Comment:
Second-degree imports like that cause Sphinx to complain, and Travis to fail. Please import functions from modules they originate from, instead of modules that import them and happen to contain them in their global namespace:
```suggestion
from fauxfactory import gen_string
```

Reply:
@omkarkhatavkar thanks for quick update.